PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,26 @@
-Type: patch
 
-Fixes #
+### PR Titles
+#### To pass testing pipeline, these need to follow Conventional Commits spec:
+https://www.conventionalcommits.org/en/v1.0.0/
+e.g.
+`fix: update broken link in NewForm component.`
+or
+`feat: create NewForm component.`
 
-## Changes proposed in this PR
 
-- Briefly describe the changes in this PR
-- Do you have some test asserting this behavior?
+### PR comment/description
+#### What is it doing?
+Tell us what it is doing from a technical perspective.
+
+#### Why is this required?
+Tell us why this is required from a business/product perspective.
+
+#### link to Jira ticket:
+Add a link to the Jira ticket.
+
+
+## Important:
+#### !!! make sure to squash merge
+
+### Note on Commits: 
+#### Currently not essential to pass through testing pipeline, but probably good to follow Conventional Commits specs anyway. These commit messages will be squash merged in the end.
```

Comment:
> but probably good to follow Conventional Commits specs anyway

I disagree; it doesn't really make sense for work-in-progress commits.

Reply:
Yeh I agree. Will change now.

Reply:
> 

Sounds good. I'll add the checkboxes.
thanks Seb